db/seeds.rb

```diff
@@ -8,12 +8,19 @@
 
 1.upto(9) do
   kudo = Kudo.create! title: Faker::Name.unique.name, 
-  content: Faker::Hacker.say_something_smart, 
-  giver: Employee.where("id = ? OR id = ?", Employee.first.id, Employee.second.id).sample, 
-  receiver: Employee.where.not("id = ? OR id = ?", Employee.first.id, Employee.second.id).sample, 
-  company_value_id: CompanyValue.all.sample[:id]
+    content: Faker::Hacker.say_something_smart, 
```

Comment:
its not an error, but I would wrap those attributes in the brackets for the sake of consistency